worker-core/src/main/java/com/github/workerframework/core/BulkWorkerThreadPool.java

```diff
@@ -86,10 +88,12 @@ private void execute()
                 = new BulkWorkerTaskProvider(task, workQueue);
 
             try {
+                activeThreads.addAndGet(1);
```

Comment:
I know it's the same but `increment` would be more normal than `add(1)`.

```suggestion
                activeThreads.incrementAndGet();
```